x/oracle/types/keys.go

```diff
@@ -18,5 +23,26 @@ const (
 )
 
 var (
-// KeyPrefixOracles defines key to store oracle
+	// OraclesKey defines key to store oracle
+	OraclesKey                 = []byte{0x01}
+	OracleRegistrationsKey     = []byte{0x02}
+	OracleRegistrationVotesKey = []byte{0x03}
+
+	IndexSeparator = []byte{0xFF}
 )
+
+func GetOracleKey(address string) []byte {
+	return append(OraclesKey, []byte(address)...)
+}
```

Comment:
Here, the address still remains as a string type.

Reply:
Oh, Thank you.
I fixed it.
https://github.com/medibloc/panacea-core/pull/364/commits/06c9238d94f32c8f97d4453a39177df512a5d609